src/components/Dashboard/FilingHistoryList/FilingTemplate.vue

```diff
@@ -1,5 +1,8 @@
 <template>
-  <v-expansion-panel class="filing-template">
+  <v-expansion-panel
+    class="filing-template py-5 px-6"
+    :disabled="true"
```

Comment:
disabling the `v-expansion-panel` resolves the issue of tabbing.

Reply:
@kialj876 @cameron-eyds @seeker25 Here is something you might use. By disabling the panel here, you prevent the user from tabbing to the panel header (which we don't want) and encourages them to tab/click to the action buttons (which we do want).